packages/oasis-actions/src/types/common/index.d.ts

```diff
@@ -0,0 +1,15 @@
+export type Address = string
```

Comment:
Could we introduce a regex as a type?

Reply:
not really but we could have we can have something like this:

[TS playground](https://www.typescriptlang.org/play?#code/C4TwDgpgBAggJnAThAziqBeKLiIJYB2A5lAGRQDeAhgsmgFxQEQBuEiAvgFBcDGA9gRxQaSVCgBKEIhAAemKAHoAegAZZAbSoBaAGYxtAMVXaAnAF0KAFlUcAJIp66ArgV7A8gqAFsqAawh4MTQAeUQAOWcAGyiACl8QACNA2nFGHHxiAEpGILp0AB8maKjKLigKqDxdKFi8FDzxeKoklOCULKyyyp6oZGBnRAIfFuTGtHLK7h7+weGCEq5uLhc3Dy9fAPGUZtbt9NxCIi6KSYqBIWARVLQFTbb8sMiY3bGbjrOq3VjRfJPPmYQAZDa7tT7TSrAAAWiH4AHcmBAEQBRRCwxCxABE4RCABVYAARAkSZEAZVJmKySygTlc7k8w3q21ivCoBDgeDgVGAEAOmWOjFZ7M53Og9Vg7wqp0BwOGv3EUhksgAdDycCy2RyuTyqcsLsJvCB9hL2nd-A8mgByS1ZIA)